doc/source/releases/20.01_announce_user.rst

```diff
@@ -8,20 +8,37 @@ January 2020 Galaxy Release (v 20.01)
 Highlights
 ===========================================================
 
-**Feature1**
-  Feature description.
+**Workflow Executions Menu**
+  This menu can be reached from the User menu -> Workflow Invocations
+  and lists your recent Workflow executions, their status and links
+  to the Workflow Editor and the History with the results of the
+  Workflow run.
```

Comment:
Could you add a screenshot of how the new view looks? Maybe showing some recent in-progress/complete/etc. ones?

Reply:
The screenhots are linked below.

Reply:
Oh, that wasn't there when I commented. I must've been on an earlier commit somehow. That looks great!